src/modules/sbstudio/plugin/operators/create_takeoff_grid.py

```diff
@@ -224,15 +184,27 @@ class CreateTakeoffGridOperator(Operator):
         update=_ensure_rows_columns_and_counts_consistent,
     )
 
-    drones_per_slot = IntProperty(
-        name="Drones per slot",
+    drones_per_slot_row = IntProperty(
+        name="Drones per slot row",
         description=(
             "Number of drones that can be placed in a single slot in the "
```

Comment:
Description needs to be updated to reflect that this is now for a single row only.